Client/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -389,7 +389,7 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
     }
 
     func loader(dataLoaded data: [Site]) {
-        self.data = data
+        self.data = data.count == 5 ? data : Array(data.dropFirst(5))
```

Comment:
I don't understand, why `dropFirst`? Also why checking == 5?

If this comes back with 7 results, you're dropping the first 5 results–resulting in 2 left–not limiting the overall list size to 5

Reply:
@kylehickinson Fixed it. I was reading the documentation wrong. I changed it to an array slice.